mleap-spark/src/test/scala/org/apache/spark/ml/parity/feature/NGramsParitySpec.scala

```diff
@@ -16,6 +16,6 @@ class NGramsParitySpec extends SparkParityBase {
     new NGram().
       setInputCol("loan_title_tokens").
       setOutputCol("loan_title_ngram").
-      setN(2))).fit(dataset)
+      setN(3))).fit(dataset)
```

Comment:
does this need to change?

Reply:
The problem here is that 2 is the default, so then it does not find the deserialization bug

Reply:
👍